City-of-Helsinki/helsinki-paatokset
# PP-234: Integrations for decisions, cases and positions of trust

**To test**
* Read code changes. Is there anything missing from the policymaker + meetings logic?
* Take a backup of your existing local environment with `drush make drush-create-dump`
* Run `make new`
* SSH into the container and run `drush ap:fs -v`
* Check that these endpoints have some data:
  * https://helsinki-paatokset.docker.so/fi/ahjo-proxy/aggregated/trustees
  * https://helsinki-paatokset.docker.so/fi/ahjo-proxy/aggregated/initiatives_all
  * https://helsinki-paatokset.docker.so/fi/ahjo-proxy/aggregated/resolutions_all
* Run the following migrations:
  * `drush mim ahjo_cases:latest`
  * `drush mim ahjo_decisions:latest`
  * `drush mim ahjo_trustees:all`
* Check that content was actually imported:
  * https://helsinki-paatokset.docker.so/fi/admin/content/cases
  * https://helsinki-paatokset.docker.so/fi/admin/content/decisions
  * https://helsinki-paatokset.docker.so/fi/admin/content/trustees
